tests/playbooks/tests_ethtool_coalesce.yml

```diff
@@ -0,0 +1,97 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- hosts: all
+  vars:
+    interface: lsrfeat1
+    type: veth
+  tasks:
+    - block:
+        - name: >-
+            TEST: I can create a profile without any coalescing option.
+          debug:
+            msg: "##################################################"
+        - import_role:
+            name: linux-system-roles.network
+          vars:
+            network_connections:
+              - name: "{{ interface }}"
+                type: "{{ type }}"
+                autoconnect: no
+                ip:
+                  dhcp4: no
+                  auto6: no
+        - name: Get profile's coalescing options
+          command: "cat /etc/sysconfig/network-scripts/ifcfg-{{ interface }}"
+          register: no_coalesce
+        - name: "ASSERT: The profile does not contain coalescing options"
+          vars:
+            e_opts: "{{ no_coalesce.stdout | regex_search('ETHTOOL_OPTS=.*') }}"
+          assert:
+            that: e_opts == ""
+        - name: >-
+            TEST: I can set rx-frames and tx-frames.
+          debug:
+            msg: "##################################################"
+        - import_role:
+            name: linux-system-roles.network
+          vars:
+            network_connections:
+              - name: "{{ interface }}"
+                type: "{{ type }}"
+                autoconnect: no
+                ip:
+                  dhcp4: no
+                  auto6: no
+                ethtool:
+                  coalesce:
+                    rx_frames: 128
+                    tx_frames: 64
+        - name: Get profile's coalescing options
+          command: "cat /etc/sysconfig/network-scripts/ifcfg-{{ interface }}"
+          register: with_coalesce
+        - name: Assert coalesce options set in profile
+          vars:
+            e_opts:
+              "{{ with_coalesce.stdout | regex_search('ETHTOOL_OPTS=.*') }}"
+          assert:
```

Comment:
It would be nice to match the full `ETHTOOL_OPTS` like to ensure that the syntax is correct. Also, for this it should be possible to use `grep ^ETHTOOL_OPTS` in the command above instead of `cat` to get only the necessary line.